dali/ft/filecopy.cpp

```diff
@@ -1592,12 +1592,32 @@ void FileSprayer::commonUpSlaves()
             cur.whichSlave = 0;
     }
 
-    if (options->getPropBool(ANnocommon, true) || pushWhole)
+    if (pushWhole)
+        return;
+
+    // noCommon is defaulted to on for non-containerized (revisit!)
+    bool noCommon = options->getPropBool(ANnocommon, !isContainerized());
+    if (isContainerized() && noCommon)
+    {
+        IWARNLOG("Ignoring noCommon option in containerized mode");
+        noCommon = false;
```

Comment:
has no effect (variable not used after this point) - coverity may complain.

Maybe restructure as:

```
    if (noCommon)
    {
        if (!isContainerized())
            return;
        IWARNLOG("Ignoring noCommon option in containerized mode");
    }
```

This does mean small sprays on bare-metal may use lots of processes, but that will be dealt with in a later PR.

Reply:
Will change noCommon test.

>This does mean small sprays on bare-metal may use lots of processes, but that will be dealt with in a later PR.

yes, no change there, but should be changed when larger refactoring is done.